storm-client/src/jvm/org/apache/storm/messaging/netty/Client.java

```diff
@@ -163,6 +166,15 @@
             waitStrategy = ReflectionUtils.newInstance(clazz);
         }
         waitStrategy.prepare(topoConf, WaitSituation.BACK_PRESSURE_WAIT);
+
+        totalConnectionAttempts = stormMetricRegistry.meter("send-connection-reconnects-" + host + ":" + port, topologyId,
```

Comment:
In other places in the code base, we prefix with `__` to indicate it is an internal metric. So I think we should probably keep following that and keep using `__send-iconnection` prefix here.